submissions/onyinyeChukwuka/css/sign-up.css

```diff
@@ -0,0 +1,247 @@
+body {
+    font-family: 'Montserrat', sans-serif;
+    background-color: #ffffff;
+}
+
+.header {
+    justify-content: space-between;
+    padding: 2px;
+    background-color: #0066f5;
+}
+
+nav {
+    padding: 20px;
+}
+
+.left-side {
+    display: flex;
+}
+
+nav>.right-side>div {
+    margin-right: 100px;
+    font-size: 16px;
+    color: #ffffff;
+    line-height: 1.19;
+    text-align: left;
+}
+
+.tab {
+    color: #ffffff;
+    text-decoration: none;
+}
+
+.link-wrapper {
+    height: 22px;
+}
+
+.link-wrapper a {
+    color: #ffffff;
+    text-decoration: none;
+}
+
+.sidebar h4 {
+    font-size: 50px;
+    font-weight: bold;
+    text-align: left;
+    line-height: 1.22;
+    color: #0066f5;
+    margin-bottom: 23px;
+    margin-top: 175px;
+}
+
+.sidebar p {
+    font-family: Montserrat;
```

Comment:
@Onyinye101 Remove  the font family here. You have declared it in the body tag already which cuts across all the html elements within it.